utils.py

```diff
@@ -151,7 +151,7 @@ def sinkable_props_for_provider(name):
         return ["source_image", "volume_name", "cpus", "memory", "disk_size"]
 
     if name == "azure":
-        return ["vm_size", "offer", "sku", "version"]
+        return ["vm_size", "offer", "sku", "version", "authorized_keys_file", "public_key_file"]
```

Comment:
I guess that this feature will be expanded to other cloud providers in the future, so having in this `azure` statement might not make sense.
Anyway, by now it shouldn't be a problem